doc/modules/cassandra/pages/architecture/snitch.adoc

```diff
@@ -2,17 +2,17 @@
 
 In cassandra, the snitch has two functions:
 
-* it teaches Cassandra enough about your network topology to route
+* It teaches Cassandra enough about your network topology to route
```

Comment:
Generally, with a colon in the previous sentence, you use lowercase to start the bullet item.